content/en/blog/_posts/2023-08-04-kubernetes-1.28-blog.md

```diff
@@ -0,0 +1,299 @@
+
+This builds on the capabilities of the CRD Validation Rules feature that graduated to beta in 1.25 but with a focus on the policy enforcement capabilities of validating admission control.
+    
+This will lower the infrastructure barrier to enforcing customizable policies as well as providing primitives that help the community establish and adhere to the best practices of both K8s and its extensions.
+
+## CRD validation ratcheting
+    
+This Allows custom resources to fail validation if the patch did not alter any of the invalid fields.  The ability to shift left validation logic from controllers to the front end is a long-term goal for improving the useability of the Kubernetes project.
+  
+## Match conditions for admission webhooks
+    
+Kubernetes v1.27 lets you specify _match conditions_ for admission webhooks,
+which lets you narrow the scope of when Kubernetes makes a remote HTTP call at admission time.
+The `matchCondition` field for ValidatingWebhookConfiguration and MutatingWebhookConfiguration
+is a CEL expression that must evaluate to true for the admission request to be sent to the webhook.
+
+In Kubernetes v1.28, that field moved to beta, and it's enabled by default.
+    
+To learn more, see [`matchConditions`](/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-matchconditions) in the Kubernetes documentation.
+
+## Changes to supported skew between control plane and node versions
+    
+This enables testing and expanding the supported skew between core node and control plane components by one version from n-2 to n-3, so that node components (kubelet and kube-proxy) for the oldest supported minor version work with control plane components (kube-apiserver, kube-scheduler, kube-controller-manager, cloud-controller-manager) for the newest supported minor version.
+    
+The Kubernetes yearly support period already makes annual upgrades possible. Users can upgrade to the latest patch versions to pick up security fixes and do 3 sequential minor version upgrades once a year to "catch up" to the latest supported minor version.
+
+However, since the tested/supported skew between nodes and control planes is currently limited to 2 versions, a 3-version upgrade would have to update nodes twice to stay within the supported skew.
+
+## Beta support for enabling swap space on Linux
+    
+This adds swap support to nodes in a controlled, predictable manner so that Kubernetes users can perform testing and provide data to continue building cluster capabilities on top of swap.
+
+There are two distinct types of users for swap, who may overlap:
+
+- Node administrators, who may want swap available for node-level performance tuning and stability/reducing noisy neighbor issues.
+    
+- Application developers, who have written applications that would benefit from using swap memory. There are hence a number of possible ways that one could envision swap use on a node.
```

Comment:
This seems unnecessary: "There are hence a number of possible ways that one could envision swap use on a node."

Reply:
Based on https://github.com/kubernetes/website/pull/42381/files#r1293746761 I've made that into a GUI-committable fix
```suggestion
- Application developers, who have written applications that would benefit from using swap memory.
```